LayoutTests/fast/events/touch/ios/touch-event-regions-layer-tree/mousemove-mouseup.html

```
@@ -0,0 +1,26 @@
+<!DOCTYPE html><!-- webkit-test-runner [ MouseEventsSimulationEnabled=true ] -->
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)